libparsec/crates/client/src/client/organization_info.rs

```diff
@@ -34,3 +36,35 @@ pub(super) async fn organization_info(
         }
     }
 }
+
+#[derive(Debug, thiserror::Error)]
+pub enum ClientGetOrganizationBootstrapDateError {
+    #[error(transparent)]
+    Internal(#[from] anyhow::Error),
+    #[error("Component has stopped")]
+    Stopped,
+    #[error("Bootstrap date not found")]
+    BootstrapDateNotFound,
+}
+
+impl From<CertifStoreError> for ClientGetOrganizationBootstrapDateError {
+    fn from(value: CertifStoreError) -> Self {
+        match value {
+            CertifStoreError::Stopped => Self::Stopped,
+            CertifStoreError::Internal(err) => err.into(),
+        }
+    }
+}
+
+pub(super) async fn get_organization_bootstrap_date(
+    client: &super::Client,
+) -> Result<DateTime, ClientGetOrganizationBootstrapDateError> {
+    client
```

Comment:
It may seem obvious now, but I would describe the strategy to get bootstrap date
```suggestion
    // The organization bootstrap date coincides with the date the first user was created. 
    // Since the first user is the only one not invited by another user, the strategy here is to 
    // look for this user and get its creation date.
    client
```